cmd/kubernetes-static: fixes and improvements (#148)
newrelic/nri-kubernetes: cmd/kubernetes-static/readme.md

## How it works

> The files in the `./data` folder are saved outputs from KSM and various kubelet endpoints, all required to run the full integration.
> The program wil start a temporary HTTP server and serve these files. The groupers are configured to use these endpoints instead of discovering them.

Reviewer: ```suggestion
The files in the `./data` folder are saved outputs from KSM and various kubelet endpoints, which will be embedded at build-time and then served from a temporary HTTP. The groupers are configured to use these endpoints instead of discovering them.
```
